scu-de/test/Manual/fiskaltrust.Middleware.SCU.DE.Test.Launcher/Program.cs

```diff
@@ -212,6 +216,19 @@ private static void ConfigureSwissbitCloud(PackageConfiguration queue, ServiceCo
             };
             bootStrapper.ConfigureServices(serviceCollection);
         }
+
+        private static void ConfigureSwissbitCloudV2(PackageConfiguration queue, ServiceCollection serviceCollection)
+        {
+            queue.Configuration.Add("TseSerialNumber", "fd79e44187bce2e2dcc886c89bf993df26d157503c4d953557b2e5af73571876");
```

Comment:
We should read that keys from the config I think so that this works as expected when configuring something different in the portal 🤔